iris_grib/tests/unit/load_convert/test_vertical_coords.py

```diff
@@ -173,5 +174,20 @@ def test_height_with_second_fixed_surface(self):
         self.assertEqual(metadata, expected)
 
 
+class TestRealData(tests.IrisGribTest):
+    def test_hybrid_height(self):
+        """
+        Temporary test for development.  Replace with proper test  or tidy up
+        for public consumption before making final PR.
+        """
+        import os.path
+        here_dirpath = os.path.dirname(__file__)
+        filepath = here_dirpath + '/../../../test_data/'
+        fname = 'faked_sample_hh_grib_data.grib2'
+        cube = load_cube(filepath + fname, 'air_temperature')
+        print(cube)
```

Comment:
Replace print statements with an actual test.
I think it's enough to just check it *has* a factory of the expected type.
Suggest : `self.assertIsInstance(cube.aux_factories[0], HybridHeightFactory)` 
( and import the factory class somewhere above )